src/test/java/seedu/address/storage/JsonAdaptedTaskTest.java

```diff
@@ -24,6 +25,7 @@ public class JsonAdaptedTaskTest {
     private static final String VALID_DESCRIPTION = BENSON.getDescription().toString();
     private static final String VALID_NOTE = BENSON.getNote().toString();
     private static final Deadline TEST_DEADLINE = new Deadline();
```

Comment:
Should this be changed to:

```suggestion
    private static final Deadline TEST_DEADLINE = Deadline.now();
```

Reply:
Noted and changed.